sds_data_manager/lambda_code/efs_lambda/lambda_function.py

```diff
@@ -0,0 +1,140 @@
+import os
+import shutil
+
+import boto3
+
+# Define the paths
+mount_path = os.environ.get("EFS_MOUNT_PATH")
```

Comment:
Should we hard-code these across the project and specify something like "The SPICE kernels are always accessed from `/mnt/spice`? I don't see a huge benefit to making this flexible and could just cause other referencing issues if group permissions are set up differently because it changes or something like that.

Reply:
I could do that.

Reply:
did that in upcoming changes